validator/accounts/wallet/wallet.go

```diff
@@ -274,7 +290,7 @@ func (w *Wallet) InitializeKeymanager(ctx context.Context, cfg iface.InitKeymana
 		if err != nil {
 			return nil, errors.Wrap(err, "could not initialize derived keymanager")
 		}
-	case keymanager.Remote:
+	case keymanager.Remote: //option should not work for --web
```

Comment:
What's up with this comment?

Reply:
the only types that work with the UI are keymanager types that are native to prysm. the UI doesn't support remote signer types as it defeats the purpose of onboarding ( importing your keys to prysm) and creating the prysm wallet.